settings.php

```diff
@@ -82,6 +82,16 @@
             )
         );
 
+        $settings->add(
+            new admin_setting_configtext(
+                'tool_excimer/cron_sample_threshold',
+                get_string('cron_sample_threshold', 'tool_excimer'),
+                get_string('cron_sample_threshold_desc', 'tool_excimer'),
+                '2',
```

Comment:
Can you convert this into a seconds based limit as the number of samples isn't easy to calculate in your head, and also when a later issue lands the sampling ratio over time may change for a given profile. The default can be reasonably large like 5 minutes